src/main/java/github/scarsz/discordsrv/api/events/AchievementMessagePreProcessEvent.java

```diff
@@ -29,12 +30,22 @@
 
     @Getter @Setter private String achievementName;
     @Getter @Setter private String channel;
-    @Getter @Setter private String message;
```

Comment:
the getter and setter for this should be deprecated, see https://github.com/DiscordSRV/DiscordSRV/blob/develop/src/main/java/github/scarsz/discordsrv/api/events/DeathMessagePreProcessEvent.java#L52-L62

Reply:
Should be added, let me know if it is done correctly